payone-adapter/src/main/java/com/commercetools/sunrise/payment/payone/PayonePaymentServiceProviderImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.commercetools.sunrise.payment.payone;
+
+import com.commercetools.sunrise.payment.domain.CreatePaymentDataProvider;
+import com.commercetools.sunrise.payment.domain.PaymentServiceProvider;
+import com.commercetools.sunrise.payment.utils.PaymentPropertiesLoadingHelper;
+import io.sphere.sdk.payments.Payment;
+import io.sphere.sdk.payments.PaymentMethodInfo;
+import io.sphere.sdk.payments.PaymentStatus;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Created by mgatz on 7/18/16.
+ */
+public class PayonePaymentServiceProviderImpl implements PaymentServiceProvider {
+
+    private PaymentPropertiesLoadingHelper propertiesLoadingHelper;
+
+    public PayonePaymentServiceProviderImpl() {
+        propertiesLoadingHelper = PaymentPropertiesLoadingHelper.createFromResource("methods/payone.properties");
```

Comment:
discussing whole property stuff currently with @butenkor 
will possibly changed from properties to JSON incl. (de)serialization and a config class

https://github.com/commercetools/commercetools-sunrise-java-payment/pull/9/files/e779eeb58ac6c41588eb4e636a9d69518d89daa9#r72029269